artemis-core-client/src/main/java/org/apache/activemq/artemis/core/protocol/core/impl/RemotingConnectionImpl.java

```diff
@@ -408,10 +416,15 @@ public void endOfBatch(Object connectionID) {
    }
 
    private void doBufferReceived(final Packet packet) {
+      if (isHealthy && !isCorrectPing(packet)) {
+         isHealthy = false;
```

Comment:
Commenting this line out will effectively disable the fix. This will cause the new test `ZeroPersistenceSymmetricalClusterTest` to fail.